### PR TITLE
Updated Zabbix version to 4.4.1

### PR DESCRIPTION
**Pull request by kulikovav**
* Updated Zabbix version to 4.4.1
* Added zabbix-java-gateway package
* Added new tests

### CODE REVIEW

File: specs/zabbix/zabbix.spec [hunk 1]
```diff
@@ -182,9 +185,33 @@ Requires(postun):     %{__service}
...
+
 %description server-pgsql
 Zabbix server with PostgresSQL database support.
 
+%package java-gateway
+Summary:              Zabbix java gateway
+Group:                Applications/Internet
+
+%if 0%{?rhel} >= 7
+Requires:             java-headless >= 1.8.0
```
Comment (andyone) [inline specs/zabbix/zabbix.spec]: Maybe is better to use java dependency for this package? We have different versions of OpenJDK packages in our repo and in this case we can't use them with this package.

File: specs/zabbix/zabbix.spec [hunk 2]
```diff
@@ -335,9 +362,13 @@ sed -i -e 's|/usr/bin/traceroute|/bin/traceroute|' database/mysql/data.sql
 sed -i -e 's|/usr/bin/traceroute|/bin/traceroute|' database/postgresql/data.sql
 sed -i -e 's|/usr/bin/traceroute|/bin/traceroute|' database/sqlite3/data.sql
 
+# change log directory for Java Gateway
+sed -i -e 's|/tmp/zabbix_java.log|%{_localstatedir}/log/zabbix/zabbix_java_gateway.log|g' src/zabbix_java/lib/logback.xml
+
 %build
 
 export PATH="/usr/pgsql-9.6/bin:$PATH"
+export GOPATH=$(pwd)
```
Comment (andyone) [inline specs/zabbix/zabbix.spec]: `GOPATH` is set but there is no `golang` in dependencies.

**Comment by kulikovav** [comment]
@andyone Thanks for noticing these things. I've updated the PR.

**Comment by andyone** [comment]
Looks good for me. Thanks!

**Comment by andyone** [comment]
Packages were built, tested and added to `kaos-testing` repository.

**Pull request merged**